integration_tests/test_search.py

```diff
@@ -578,169 +737,160 @@ def test_unowned_partial_title_rosetta_stone(self):
+    def test_marriage_lie(self):
         self.search(
             "Marriage liez",
             FirstMatch(title="The Marriage Lie")
         )
 
-    def test_misspelled_title_match_emmie(self):
-        # NOTE: this currently fails in both versions of ES.
-        # The target title does appear in the top search results,
-        # but it's not first.
-
+    def test_invisible_emmie(self):
         # One word in the title is slightly misspelled.
         self.search(
-            "Ivisible emmie",
-            FirstMatch(title="Invisible Emmie")
+            "Ivisible emmie", FirstMatch(title="Invisible Emmie")
         )
 
-    def test_misspelled_title_match_karamazov(self):
-        # NOTE: this works on ES1 but fails on ES6; not only is the target title
-        # not the first result in ES6, it's not any of the top results.  Fixing
-        # the typo makes it work.
-
+    def test_karamazov(self):
         # Extremely uncommon proper noun, slightly misspelled
         self.search(
             "Brothers karamzov",
             FirstMatch(title="The Brothers Karamazov")
         )
 
-    def test_misspelled_title_match_wave(self):
-        # NOTE: this currently fails in both versions of ES; the target book is
-        # result #4.  Fixing the typo fixes the search results.
-
+    def test_restless_wave(self):
         # One common word in the title is slightly misspelled.
         self.search(
             "He restless wave",
             FirstMatch(title="The Restless Wave")
         )
 
-    def test_misspelled_title_match_kingdom(self):
-        # NOTE: this works in ES1, but not in ES6!  In ES6, the target title is not
-        # in the top search results.
-
+    def test_kingdom_of_the_blind(self):
         # The first word, which is a fairly common word, is slightly misspelled.
+        #
+        # The desired book is not on the first page.
         self.search(
             "Kngdom of the blind",
-            FirstMatch(title="The Kingdom of the Blind")
+            FirstMatch(title="Kingdom of the Blind")
         )
 
-    def test_misspelled_title_match_husbands(self):
+    def test_seven_husbands(self):
         # Two words--1) a common word which is spelled as a different word
         # ("if" instead of "of"), and 2) a proper noun--are misspelled.
         self.search(
             "The seven husbands if evyln hugo",
             FirstMatch(title="The Seven Husbands of Evelyn Hugo")
         )
 
-    def test_misspelled_title_match_nightingale(self):
-        # NOTE: this fails in both versions of ES, but the top ES1 results are reasonable
-        # (titles containing "nightfall"), whereas the top ES6 result is entitled
-        # "Modern Warfare, Intelligence, and Deterrence."
-
-        # Unusual word, misspelled
+    def test_nightingale(self):
+        # Unusual word misspelled.
+        #
+        # This might fail because a book by Florence Nightingale is
+        # seen as a better match.
```

Comment:
Sometimes a test passes right now but I've seen it go back and forth as I tweaked the search algorithm. In that case I put a note like this one explaining why a test might _start_ failing.